WordPress/WordPressStatsWidgets/LockScreenWidgets/Views/LockScreenSingleStatView.swift

```diff
@@ -0,0 +1,34 @@
+import SwiftUI
+import WidgetKit
+
+@available(iOS 16.0, *)
+struct LockScreenSingleStatView: View {
```

Comment:
Would it be possible to add `PreviewProvider` at the bottom? It would be a nice way to see how the view reacts to different types of inputs.

Reply:
Good suggestion!! I forgot it, thanks for the reminder.